core/main/src/Core/Particles.ts

```diff
@@ -188,6 +188,31 @@ export class Particles {
             this.remove(particle);
         }
 
+        // Repulse
```

Comment:
Iterating through all particles array is a big waste, handle this with a Particles Interact like `Linker`

Reply:
Done